spec/dummy/spec/integration/user_commands_spec.rb

```diff
@@ -18,4 +18,13 @@
       expect(result.error.messages[:name]).to include("can't be blank")
     end
   end
+
+  describe 'delete' do
+    it 'deletes record' do
+      users.create.call(name: 'Piotr')
+      result = users.try { delete(:by_name, 'Piotr') }
+
+      expect(result.error).to be(nil)
```

Comment:
OK I pushed a fix and released new rom-sql. If you could rebase this PR on top of latest master I'm happy to merge it in. It is a useful spec!